frontend/src/index/App.tsx

```diff
@@ -1,14 +1,15 @@
 import React, {useEffect, useState} from 'react';
-import './css/App.css';
 import {BrowserRouter, NavLink, Route, Routes} from "react-router-dom";
-import StockOverview from "../stock/StockOverview";
 import axios from "axios";
+import StockOverview from "../stock/StockOverview";
+import HorseOverview from "../horses/HorseOverview";
 import ClipLoader from "react-spinners/ClipLoader";
 import LoginPage from '../security/LoginPage';
 import RegisterPage from "../security/RegisterPage";
 import "../buttons/css/LogoutButton.css";
 import "../buttons/css/GeneralButtonStyles.css";
-
+import "../index/css/GeneralOverviewPage.css";
+import './css/App.css';
```

Comment:
Why do you need so many different .css-files? I believe that if you have an import already somewhere else you can still access the classes

Reply:
The idea: every component only loads the css it needs. App.tsx is responsible for general layouts of the whole app so it loads quite a lot of css